x/shield/keeper/pool.go

```diff
@@ -52,7 +54,7 @@ func (k Keeper) CreatePool(
 	var endTime, endBlockHeight int64
 	startBlockHeight := ctx.BlockHeight()
 	if timeOfCoverage != 0 {
-		endTime = ctx.BlockTime().Unix() + timeOfCoverage
+		endTime = ctx.BlockHeader().Time.Unix() + timeOfCoverage
```

Comment:
What's the difference between `BlockTime()` and `BlockHeader().Time`?

Reply:
they're the same